File: webapi/src/webapi/Domain/Aggregates/GrupoEtapa/GrupoEtapa/GrupoEtapa.php
<?php
namespace webapi\Domain\Aggregates\GrupoEtapa\GrupoEtapa;


use webapi\Domain\Aggregates\Core\Entity;

/**
* @Entity
* @Table(name="grupoetapa")
*/
class GrupoEtapa extends Entity 
{
	/**
	* @var integer @id_grupoetapa
	* @Id
    * @Column(name="id_grupoetapa", type="integer")
    * @GeneratedValue(strategy="AUTO")
    */
	private $id_grupoetapa;
	
	
	/**
	*
	* @Column(name="nome", type="string" , length=30) 
	*/
	private $nome;

	public function getId_grupoetapa(){
		return $this->id_grupoetapa;
	}

	public function setId_grupoetapa($id_grupoetapa){
		$this->id_grupoetapa = $id_grupoetapa;
	}

	public function getNome(){
		return $this->nome;
	}

	public function setNome($nome){
		$this->nome = $nome;
	}
}

?>

File: Spring001/src/main/java/ba/com/grisecorp/domain/aggregates/project/grupoetapa/GrupoEtapa.java
package ba.com.grisecorp.domain.aggregates.project.grupoetapa;

import java.util.List;


import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.OneToMany;

import ba.com.grisecorp.domain.aggregates.project.programa.Programa;

@Entity(name="grupoetapa")
public class GrupoEtapa
{ 

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name="id_grupoetapa")
	private int idgrupoetapa;

	@Column(name="name")
	private String name;
	
	@OneToMany(fetch = FetchType.LAZY, mappedBy = "grupoetapa")
	private List<Programa> programas;
	
	public int getIdgrupoetapa() 
	{
		return this.idgrupoetapa;
	}
	
	public void setId_grupo_etapa(int idgrupoetapa) 
	{
		this.idgrupoetapa = idgrupoetapa;
	}
	
	public String getName() 
	{
		return this.name;
	}
	
	public void setName(String name)
	{
		this.name = name;
	}

	public List<Programa> getProgramas() {
		return programas;
	}

	public void setPrograma(List<Programa> programas) {
		this.programas = programas;
	}
	
	
}
